apps/crypto/src/App.jsx

```diff
@@ -119,8 +187,10 @@ const queryToData = async ({ query }) => {
     return;
   }
 
+  // fetchdata(coin);
+
   const response = await fetch(
-    "https://data.messari.io/api/v1/assets?fields=id,name,symbol,metrics/market_data/price_usd,metrics/market_data/ohlcv_last_24_hour",
+    `https://data.messari.io/api/v1/assets/btc/metrics/price/time-series`,
```

Comment:
A user would only be able to bitcoin prices now.

Reply:
Yes I only implemented bitcoin for now to get the ok from you for the UI.